libs/langchain-community/src/chat_models/tests/chatbedrock.int.test.ts

```diff
@@ -21,21 +21,21 @@ import { BedrockChat } from "../bedrock/web.js";
 void testChatModel(
   "Test Bedrock chat model: Claude-v2",
   "us-east-1",
-  "anthropic.claude-v2",
+  "mistral.mistral-7b-instruct-v0:2",
   "What is your name?"
```

Comment:
Shouldn't this be added as separate tests? Claude support is pretty important too :D